torch/_inductor/codecache.py

```diff
@@ -1115,7 +1117,7 @@ class CompiledFxGraph:
     mutated_input_idxs: Set[int]
     constants: Dict[str, torch.Tensor]
     torchbind_constants: Dict[str, torch._C.ScriptObject]
-    output_strides: Optional[List[Optional[Tuple[int, ...]]]]
+    output_strides: Optional[List[Optional[Tuple[sympy.Expr, ...]]]]
```

Comment:
nit nit: maybe call them output_strides_exprs? (since they're different from the output_strides that ultimately get attached to the context)

Reply:
I did this at first but it required renaming quite a few things, especially in torch/_guards.py and since python is not typed very well, i did not want to miss one. So, I chose to not do a renaming. (and mypy did not even catch the one in guards.py

Reply:
Sounds good then